MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/JitterBuffer.cs

```diff
@@ -256,6 +367,14 @@ public void step(float timestep)
 						CurrentSnapshot = SnapshotBuffer.Snapshots.First().Value;
 						CurrentLocalTime = CurrentSnapshot.Time;
 					}
+					else
+					{
+						if (_mode != Mode.Init)
+						{
+							CurrentSnapshot = SnapshotBuffer.Snapshots.First().Value;
+							CurrentLocalTime = CurrentSnapshot.Time;
+						}
```

Comment:
Yes, this was my intention, but I guess this will have other unwanted consequences.
My intention was to put the latest snapshot after a jitter buffer reset to the latest update, and without this the jitter buffer did not gave me any update for 1-2 secs.
Do you know any better solutions?

---
In reply to: [441584989](https://github.com/microsoft/mixed-reality-extension-unity/pull/189#discussion_r441584989) [](ancestors = 441584989)

Reply:
now the buffer  is in init mode after reset

---
In reply to: [441835897](https://github.com/microsoft/mixed-reality-extension-unity/pull/189#discussion_r441835897) [](ancestors = 441835897,441584989)